index.js

```diff
@@ -445,6 +506,12 @@ async function onGithubPullRequest(payload) {
       }
       await autoMergePullRequest(repoName, prNumber);
     }
+    for (let index = 0; index < customLabels.length; ++index) {
```

Comment:
Let's merge this block and 487-492 into a function, it's the exact same code I think